docs/xplat-implementation.md

```diff
@@ -7,17 +7,30 @@ title: Cross Platform Implementation
 
 #### The React Native renderer utilizes a core render implementation to be shared across platforms
 
-In the previous render system of React Native, the _React Shadow Tree_, _Layout System_, and [View Flattening](view-flattening.md) algorithm were implemented once for each platform. The React Native render system was designed to be a cross-platform solution which enables the implementation of all features to be shared across all platforms. The React Native team intends to incorporate an animation system into the render system and also extend the Fabric render system to new platforms such as Windows, PlayStation, and more.
+In the previous render system of React Native, the **[React Shadow Tree](glossary#react-shadow-tree-and-react-shadow-node)**, layout logic, and **[View Flattening](view-flattening.md)** algorithm were implemented once for each platform. The current renderer was designed to be a cross-platform solution by sharing a core C++ implementation.
+
+<!--TODO: Does the below paragraph makes sense to keep?-->
```

Comment:
I think if we expand it a little bit and add link to many platform vision, it is worth keeping. I've noted this and will address it later.